tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -558,19 +558,30 @@ Dashboard Test Teardown
     Close All Browsers
 
 Check GPU Resources
-    ${version-check}=    Is RHODS Version Greater Or Equal Than    1.9.0
-    IF    ${version-check}==True
-        ${elements}=    Get WebElements    //article
-        ${len}=    Get Length    ${elements}
-        Should Be Equal As Integers    ${len}    1
-        Page Should Contain Element    //article[@id="python-gpu-numba-tutorial"]
-        Page Should Contain Element    //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
+    [Documentation]   Check resource tiles for GPU is present
+    ${version_check}=    Is RHODS Version Greater Or Equal Than    1.13.0
+    ${elements}=    Get WebElements    //article
+    @{gpu_re_id}=    Create List  'gpu-enabled-notebook-quickstart'   'python-gpu-numba-tutorial'
+    ...    'gpu-quickstart'     'nvidia-doc'
+    @{gpu_re_link}=   Create List   '#'  'https://github.com/ContinuumIO/gtc2018-numba'   '#'
+    ...   'https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/contents.html'
+    ${len}=    Get Length    ${elements}
+    IF    ${version_check}==True
+        Should Be Equal As Integers    ${len}    4
+        FOR    ${counter}    IN RANGE    ${len}
+           Page Should Contain Element    //article[@id=${gpu_re_id}[${counter}]]
+           IF    ${gpu_re_link}[${counter}] == '#'
+                 ${counter}=    Get WebElements   //a[@href=${gpu_re_link}[${counter}]]
+                 ${no_of_open_link}=    Get Length    ${counter}
+                 Run Keyword IF   ${no_of_open_link} == ${2}   Log   There are two tile with `Open' link
```

Comment:
why this line?

Reply:
There are two tile which href attributes is common ( '#') so to make sure that we have both of them i had to add that

Reply:
ah, got it. that's fine. Maybe, instead of fixing 2, you can count the `#` from the list. It would avoid missing to update the fixed number if a new quick start will be added in the future.

Reply:
I see you point. But if any changes are happening in adding or removing of the tiles  we should be aware of it. keeping that in mind i think it is good idea to keep it as a fixed number

Reply:
Agree. Changes are done

Reply:
cool, thx